java/client/src/main/java/glide/api/commands/SetBaseCommands.java

```diff
@@ -221,4 +221,39 @@ public interface SetBaseCommands {
      * }</pre>
      */
     CompletableFuture<Long> sunionstore(String destination, String[] keys);
+
+    /**
+     * Returns a random element from the set value stored at <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/srandmember/">redis.io</a> for details.
+     * @param key The key from which to retrieve the set member.
+     * @return A random element from the set, or <code>null</code> if <code>key</code> does not exist.
+     * @example
+     *     <pre>{@code
+     * client.sadd("test", new String[] {"one"}).get();
+     * String response = client.srandmember("test").get();
+     * assertEquals("one", response);
+     * }</pre>
+     */
+    CompletableFuture<String> srandmember(String key);
+
+    /**
+     * Returns an array of distinct random elements from the set value stored at <code>key</code> if a
```

Comment:
```suggestion
     * Returns multiple random elements from the set value stored at <code>key</code> if a
```

Reply:
Changing this to just `random elements from ...` since saying "multiple" feels like there will always be more than one results

Reply:
Rigth! Or `one or more`